moveit_experimental/jog_arm/src/jog_arm/jog_calcs.cpp

```diff
@@ -124,16 +124,12 @@ JogCalcs::JogCalcs(const JogArmParameters parameters, JogArmShared& shared_varia
   // Now do jogging calcs
   while (ros::ok())
   {
-    // If user commands are all zero, reset the low-pass filters when commands resume
+    // Flag that incoming commands are all zero. May be used to skip calculations/publication
     pthread_mutex_lock(&mutex);
     bool zero_cartesian_cmd_flag = shared_variables.zero_cartesian_cmd_flag;
     bool zero_joint_cmd_flag = shared_variables.zero_joint_cmd_flag;
```

Comment:
These variables are unused, should remove.

Reply:
It looks like they are used (e.g. lines 148,206). But now that I think about it, there are 2 mutex locks here that can be combined.